### PR TITLE
fix(eslint-config-angular): support `no-useless-rename`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/eslint-config-angular/internal/base.js [hunk 1]
```diff
@@ -31,5 +31,13 @@ module.exports = {
     'no-param-reassign': 'off',
     'no-case-declarations': 'error',
     'no-console': ['error', { allow: ['info', 'assert', 'warn', 'error'] }],
+    'no-useless-rename': [
+      'error',
+      {
+        ignoreDestructuring: true,
+        ignoreImport: false,
```
Comment (SuperOleg39) [inline packages/eslint-config-angular/internal/base.js:38]: А почему оставляем ignoreImport?

Comment (splincode) [reply packages/eslint-config-angular/internal/base.js:38]: Because we give it to typescript eslint check

Comment (splincode) [reply packages/eslint-config-angular/internal/base.js:39]: @SuperOleg39 what do you think? I need fix this PR?

**Review by SuperOleg39 — APPROVED** [review]
LGTM

**Pull request merged**